packages/intuit/src/Utils.js

```diff
@@ -0,0 +1,45 @@
+import { composeNextState } from '@openfn/language-common';
+import {
+  request as commonRequest,
+  logResponse,
+} from '@openfn/language-common/util';
+
+export const prepareNextState = (state, response) => {
+  const { body, ...responseWithoutBody } = response;
+
+  if (!state.references) {
+    state.references = [];
+  }
+
+  return {
+    ...composeNextState(state, response.body),
+    response: responseWithoutBody,
+  };
+};
+
+export const request = (state, method, path, options = {}) => {
+  const { baseUrl, access_token } = state.configuration;
+
+  const {
+    query = {},
+    body = {},
+    headers = { 'Content-type': 'application/json' },
+    parseAs = 'json',
+  } = options;
+
+  const opts = {
+    parseAs,
+    baseUrl,
+    body,
+    query,
+    headers: {
+      Accept: 'application/json',
```

Comment:
Why do we default `accept` here, but `content-type` in the deconstruction above?

If we're going to default headers ,we should do it all in the same place. Nice and tidy.